### PR TITLE
Add senior product designer role

### PR DESCRIPTION
**Pull request by josh-padnick**
- Update careers page to list benefits, discuss remote-first nature, and sane working hours
- Add senior product designer role
- Consolidate X / Senior X / Principal X titles

**Preview: See https://5f04ea1976b1d1000979908f--keen-clarke-470db9.netlify.app/careers/**

Also, note that I have the designer role applying to Workable because I initially ran into some issues with applicants applying to Breezy directly. Note that we have not made a decision to move off of Workable, so all other jobs should either apply via careers@gruntwork.io or to Breezy.

### CODE REVIEW

**Comment by josh-padnick** [comment]
Thanks! I'm going to merge this now but happy to make additional changes if needed.

**Comment by oredavids** [comment]
@josh-padnick My initial impression of the Apply CTA was that it was meant for the designer role. What do you think about either adding a horizontal line (like we already use to separate the different roles) just before the "Apply Now" Button or use some spacing?

Screen Shot 2020-07-07 at 5 53 22 PM

**Comment by josh-padnick** [comment]
That "apply now" actually is for the designer role! I'm not sure I understood your other suggestions; can you clarify what effect ou were trying to achieve?

**Comment by oredavids** [comment]
Ohh I understand now. I wrongly thought the button was meant to be a generic CTA for all the open positions.

**Comment by josh-padnick** [comment]
Ah, gothca. Well now I'm wondering if others will see it that way. Any thoughts on what we could do to clarify its intent?

File: pages/careers/_human-friendly.html [hunk 1]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Dental Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>Vision Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>FSA and HSAs.</strong> We don't contribute to these accounts, but we do offer them as an option.</li>
+      <li><strong>401(k).</strong> We contribute 3% of your salary to your 401(k), regardless of how much you contribute yourself.</li>
+      <li><strong>Disability insurance.</strong> If you get disabled, we have a policy that will pay out a portion of your salary.</li>
+      <li><strong>Hardware budget.</strong> We'll buy you a brand new 16" Apple MacBook Pro upon joining. It will be owned by you, not the company.</li>
+      <li><strong>Personal Budget.</strong> We'll give you a personal budget of $500/month to spend on improving your health, productivity, or knowledge.</li>
+      <li><strong>Minimum Vacation.</strong> We require that you take at least 4 weeks of vacation per year.</li>
```
Comment (brikis98) [inline pages/careers/_human-friendly.html:34]: "We have a minimum vacation policy <link to relevant page that explains more> where every employee is required to take <em>at least</em>"

File: pages/careers/_human-friendly.html [hunk 2]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+    </ul>
+
+    <p><em>The above benefits reflect our offering package for Grunts based in the USA. For Grunts based in other countries, we make every effort to offer an equivalent total package specific to that country.</em></p>
```
Comment (brikis98) [inline pages/careers/_human-friendly.html:37]: This makes all non US employees feel like second class citizens. Remove. Instead, update the above descriptions equivalently.

File: pages/careers/_human-friendly.html [hunk 3]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+
+    <h3>100% Remote</h3>
+    <p>Gruntwork is a remote-first company.</p>
```
Comment (brikis98) [inline pages/careers/_human-friendly.html:40]: Expand a bit more: "Gruntwork is a remote-first company. That means you can work from home, a co-working space, cafe, or wherever else you prefer, there's no commute, and you have complete control over your work hours. To allow at least a few hours of overlap between all employees for the occasional live meeting or pair-coding session, we are currently hiring from most countries in the time zones between GMT-7 (e.g., California, USA) to GMT+2 (e.g., Berlin, Germany)."

File: pages/careers/_human-friendly.html [hunk 4]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+
+    <h3>Sane Working Hours</h3>
+    <p>Although we're a distributed team, we only hire candidates between the time zones of GMT-7 (San Francisco) and GMT+2 (Berlin). This means you'll never be separated by more than 9 hours from your colleagues and never pressured to work at crazy hours just to get your work done.</p>
```
Comment (brikis98) [inline pages/careers/_human-friendly.html:43]: A 9-hour span actually does sound like you'd be pressured to occasionally work weird hours, so I'm not sure this section is that valuable. 

If you want to have it, focus instead on:

1. Remote work lets you control your hours and schedule work around life.
1. In general, we try to stick to normal working hours: no more than ~40h per week, and no working on weekends or holidays.
1. We have a minimum vacation policy: must take at least 4 weeks per year away from work.
1. We don't have an on-call rotation, no one carries a pager, our support contracts have SLAs of responses on business days / hours only.

File: pages/careers/_open-positions.html [hunk 5]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
-      positions, but with the added expectation that you take more ownership,
-      get (complex) things done, and communicate even more effectively.
-    </p>
+    <a target="senior-product-designer"></a>
+    <h3>Senior Product Designer</h3>
 
-    <hr />
+    <p>Help us make Gruntwork delightful!</p>
+
+    <h4>What's Gruntwork?</h4>
```
Comment (brikis98) [inline pages/careers/_open-positions.html:206]: Why would solely the designer job description have this section? IMO, this content is described better in many other places on the website, and this should be removed.

File: pages/careers/_open-positions.html [hunk 6]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+       "Gruntwork" because we take away this Gruntwork for our clients so that they can focus on the parts of their software
+       that are unique to them.</p>
+    <p>We are 100% remote, ~15 people, profitable, growing (even during COVID!) and fully bootstrapped with no debt and
+       no investors. Our small company works with both startups and name companies like Toyota, Adobe, and the US House
+      of Representatives. We also <a href="#gruntwork-is-a-human-friendly-company">care a lot about humans.</a></p>
+
+    <h4>What's the opportunity?</h4>
+    <ul>
+      <li>
+        <strong>Product Design.</strong> We're building a SaaS platform that will become the primary customer experience of our product. Today, customers access our product as private GitHub repos plus a patchwork of links to training and documentation. Working with you, we envision a portal where customers login to discover and use all the elements of our existing product, brand it to themselves, and ultimately offer a UI-based way to learn, deploy and manage infrastructure in the cloud, all backed by infrastructure as code under the hood.
```
Comment (brikis98) [inline pages/careers/_open-positions.html:219]: I'd separate the portal from self-service. These are separate products and separate opportunities. The portal is all about a better experience for accessing our existing product: the IaC Lib, training, support, docs, etc. Self service will be a completely new product/experience to design that solves the problem of "big shiny deploy button" for most of the company and IaC under the hood for the Ops people.

File: pages/careers/_open-positions.html [hunk 7]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+      </li>
+      <li>
+        <strong>Branding and Marketing Website.</strong> We have an improved understanding of our business that we'd like to reflect in a new public marketing website. We'll look to you lead the design effort here.
```
Comment (brikis98) [inline pages/careers/_open-positions.html:222]: This is too narrow of a take. Focus on something more like, "You'd be the owner of Gruntwork's public face to the world. How do we tell the world what we do? How do we explain and showcase our highly technical products in a way that visitors can understand? How do we convert visitors to leads?"

File: pages/careers/_open-positions.html [hunk 8]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+    <p>We give software teams a complete, world-class cloud infrastructure that lets them go to prod in days. Typically,
+       the process of launching prod infrastructure in the cloud is awful. It takes months of tedious work, hundreds of
+       thousands of dollars, and often results in poorly maintained code and anti-patterns in the end. We call our company
+       "Gruntwork" because we take away this Gruntwork for our clients so that they can focus on the parts of their software
+       that are unique to them.</p>
+    <p>We are 100% remote, ~15 people, profitable, growing (even during COVID!) and fully bootstrapped with no debt and
+       no investors. Our small company works with both startups and name companies like Toyota, Adobe, and the US House
+      of Representatives. We also <a href="#gruntwork-is-a-human-friendly-company">care a lot about humans.</a></p>
+
+    <h4>What's the opportunity?</h4>
```
Comment (brikis98) [inline pages/careers/_open-positions.html:216]: This title should be "What you'll work on" to be consistent with our other job roles.

File: pages/careers/_open-positions.html [hunk 9]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+        <strong>Branding and Marketing Website.</strong> We have an improved understanding of our business that we'd like to reflect in a new public marketing website. We'll look to you lead the design effort here.
+      </li>
+      <li>
+        <strong>Design system.</strong> You'll be our first full-time designer so you will set the design direction for the entire company across all areas of design, working directly with the founders and closely with engineers.
+      </li>
+    </ul>
+
+    <p>Although you'll be the only professional designer on the team, our company cares deeply about design so you'll have many eager collaborators when it comes time to brainstorm.</p>
+
+    <h4>What skills are we looking for?</h4>
```
Comment (brikis98) [inline pages/careers/_open-positions.html:231]: This title should be "Your ideal background" to be consistent with our other job roles.

File: pages/careers/_open-positions.html [hunk 10]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong> When we present our product vision, we'll look to you to offer opinions, insights, and methodologies to help us translate that vision into a fantastic product.
```
Comment (brikis98) [inline pages/careers/_open-positions.html:235]: Might want to say something along the lines of, "We are looking for a product designer—someone who can be an equal partner in helping us design and develop our products..."

File: pages/careers/_open-positions.html [hunk 11]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+      </li>
+      <li>
+        <strong>Design skill.</strong> At the end of the day, the designs you create should look great and be easy to use. Our hope is that you can make our users smile.
+      </li>
+      <li>
+        <strong>Domain expertise.</strong> We don't expect you to have a complete vision for our product on day 1, but we're looking for someone who either already understands our domain (DevOps, CI/CD, Infrastructure as Code, AWS), or who feels comfortable ramping up on it.
```
Comment (brikis98) [inline pages/careers/_open-positions.html:241]: I think I'd rephrase this as, "We don't expect you to have a complete vision for our product on day 1, but to be able to design products in our space, you must be able to thoroughly ramp up on our domain (DevOps, software delivery, cloud software, CI / CD, infrastructure as code). Prior experience software engineering, working on dev tools, and other highly technical domains is a big plus."

File: pages/careers/_open-positions.html [hunk 12]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+      </li>
+      <li>
+        <strong>Bonus: HTML/CSS.</strong> If you can roll up your sleeves and review our engineers' work in HTML/CSS, you can help us make sure the final product is true to your vision.
```
Comment (brikis98) [inline pages/careers/_open-positions.html:244]: Review? Or write?

File: pages/careers/_open-positions.html [hunk 13]
```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
...
+      </li>
+      <li>
+        <strong>Bonus: Engineering Background.</strong> If you happen to have an engineering background such as previous experience coding an app or a computer science degree, that may give you additional insight into how our product might especially delight users.
+      </li>
+    </ul>
+
+    <h4>What time zones do we work in?</h4>
+    <p>While Gruntwork <a href="#sane-working-hours">generally</a> hires anywhere between San Francisco and Berlin, for this role, you'll be working with a team based in the USA/Canada (GMT-7 to GMT-3). We prefer to hire candidates located in the USA or Canada, but are open to hiring in any country within those time zones. Unfortunately, this role is not available outside the GMT-7 to GMT-3 time zones.</p>
+
+    <a href="https://apply.workable.com/gruntwork-io/j/03345F97D6/apply/" class="btn btn-primary-hollow btn-sm d-inline-block btn-dpa"" role="button">Apply Now</a>
```
Comment (brikis98) [inline pages/careers/_open-positions.html:254]: Apply now button should be below each job role, and not just this one.

File: pages/careers/_human-friendly.html [hunk 14]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
     we built a distributed, self-funded, family-friendly, profitable startup</a> and
     <a href="https://blog.gruntwork.io/how-we-got-to-1-million-in-annual-recurring-revenue-with-0-in-fundraising-340ed2b4e158" target="_blank">How
-    we got to $1 million in annual recurring revenue with $0 in fundraising</a> for all the details on what we do,
-    how we fund it, how we hire, and how we work.
+    we got to $1 million in annual recurring revenue with $0 in fundraising</a> for all the details on what we do, how we fund it, how we hire, and how we work.</p>
+
+    <h3>Benefits</h3>
+    <p>Our benefits reflect our values. We believe compensation should be fair, transparent, and generous.</p>
+    <ul>
+      <li><strong>Above-Market Salary.</strong> We compute your salary <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">formulaically</a>, using as inputs the average salary for your job title in New York City, the Gruntwork multiplier (currently 1.45) and the cost of living in your home city relative to New York City as reported by numbeo.com. We aim to systematically pay above market.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:22]: Fixed in e1d98b2.

File: pages/careers/_human-friendly.html [hunk 15]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Profit-Sharing Bonus.</strong> We set aside a pot of money at the end of each year based on profits and distribute bonuses according to a formula that uses as inputs your level within the company and the length of your tenure at the company.</li>
+      <li><strong>Performance Bonus.</strong> We give performance-based bonuses of $5,000 - $15,000 as often as once a quarter, depending on your performance.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:24]: Fixed in 1e148cc.

File: pages/careers/_human-friendly.html [hunk 16]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Progressive Equity.</strong> We grant <a href="https://medium.com/detour-dot-com/introducing-progressive-equity-f424a51ee3a4">progressive equity</a> to all new Grunts.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:25]: Fixed in 93388a8.

File: pages/careers/_human-friendly.html [hunk 17]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Medical Insurance.</strong> We offer high-quality plans, including some with no deductible through JustWorks.com. We cover 80% of your health insurance costs and 55% of those of your family.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:26]: Several people asked about the percentages. But made this more generic for now. Will leave the fully paid vision / dental in there. Clarified at the bottom this only applies to the USA and that we make every effort to match benefits.

[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:26]: Fixed in c9ec568.

File: pages/careers/_human-friendly.html [hunk 18]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Dental Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>Vision Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:28]: See https://github.com/gruntwork-io/gruntwork-io.github.io/pull/331#discussion_r452386536.

File: pages/careers/_human-friendly.html [hunk 19]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>FSA and HSAs.</strong> We don't contribute to these accounts, but we do offer them as an option.</li>
+      <li><strong>401(k).</strong> We contribute 3% of your salary to your 401(k), regardless of how much you contribute yourself.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:30]: It's relevant to salary expectations so will leave in for now. I can't imagine we'll lower this so we'd only understate. Mentioned USA vs. other below.

File: pages/careers/_human-friendly.html [hunk 20]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Disability insurance.</strong> If you get disabled, we have a policy that will pay out a portion of your salary.</li>
+      <li><strong>Hardware budget.</strong> We'll buy you a brand new 16" Apple MacBook Pro upon joining. It will be owned by you, not the company.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:32]: Fixed in 5ba4bec.

File: pages/careers/_human-friendly.html [hunk 21]
```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
...
+      <li><strong>Personal Budget.</strong> We'll give you a personal budget of $500/month to spend on improving your health, productivity, or knowledge.</li>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:33]: Fixed in 9a4b6d3.

File: pages/careers/_human-friendly.html (hunk 1 above)
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:34]: Fixed in e15c681.

File: pages/careers/_human-friendly.html (hunk 2 above)
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:37]: Ah, didn't realize it read that way. Ok, made more changes to most of the above in b47acab.

File: pages/careers/_human-friendly.html (hunk 14 above)
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/careers/_human-friendly.html:22]: A few notes:

* "formulaically" is spelled wrong. 
* "with the aim to systematically pay above market" sounds like empty promises.
* Perhaps simplify to something like: "To reduce bias and increase transparency, we compute all salaries using formulas. The formula factors in your title and location and uses a multiplier to produce a result that's above market for that title and location."

File: pages/careers/_human-friendly.html (hunk 17 above)
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/careers/_human-friendly.html:26]: Hm, if people ask, we should tell them. I don't think we need to list it on the website though. As a reminder, we already list benefits in our offer letter template and Notion page and have forgotten to update those multiple times before. Adding yet another location guarantees the data will be out of date, and potentially misleading.

File: pages/careers/_human-friendly.html (hunk 4 above)
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:43]: Good call. Reworked in c71b8d8.

File: pages/careers/_human-friendly.html (hunk 19 above)
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/careers/_human-friendly.html:30]: Again, there's no way we'll remember to keep this up to date. If people want to know, they can ask, and we can tell them. It doesn't need to live on the website though.

File: pages/careers/_human-friendly.html (hunk 21 above)
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/careers/_human-friendly.html:33]: https://github.com/gruntwork-io/gruntwork-io.github.io/commit/9a4b6d3117085d656173dc75e9468d2a34c9cf47 has the phrase "a personal budget of several hundred dollars per month to spend on" duplicated.

File: pages/careers/_open-positions.html (hunk 6 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:219]: Well, I didn't want to give away too much of our plans on this public page. I think for the designer context, combining them is probably ok.

File: pages/careers/_human-friendly.html (hunk 2 above)
Comment (brikis98) [reply pages/careers/_human-friendly.html:37]: If this is supposed to apply to multiple countries, having exact numbers is even more inaccurate / misleading. If you remove those numbers, then it's more honest and easier to maintain!

File: pages/careers/_open-positions.html (hunk 6 above)
Comment (brikis98) [reply pages/careers/_open-positions.html:219]: We had that very description on the Houston page for months! It's not giving anything away.

File: pages/careers/_open-positions.html (hunk 13 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:254]: Already addressed in #332.

File: pages/careers/_open-positions.html (hunk 7 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:222]: Good call. Fixed in 629e840.

File: pages/careers/_open-positions.html (hunk 8 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:216]: Fixed in 629e840.

File: pages/careers/_open-positions.html (hunk 9 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:231]: Fixed in 629e840.

File: pages/careers/_open-positions.html (hunk 10 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:235]: Updated in 629e840.

File: pages/careers/_open-positions.html (hunk 11 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:241]: Added in 629e840.

File: pages/careers/_open-positions.html (hunk 12 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:244]: Fixed in 629e840.

File: pages/careers/_human-friendly.html (hunk 14 above)
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:22]: Ah that does sound better. Fixed in c4d8a0f.

File: pages/careers/_human-friendly.html (hunk 21 above)
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:33]: Fixed in 8cf729aa5fbf5b3cea1619878cbe5287de4f7d9e.

File: pages/careers/_human-friendly.html (hunk 2 above)
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:37]: Ok, you seem to feel pretty strongly about that and agreed we don't want these numbers to get out of date so fixed in 150196cd4140a538d19d84a38b793b0cbfa64d2a.

File: pages/careers/_human-friendly.html (hunk 3 above)
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:40]: Added.

File: pages/careers/_open-positions.html (hunk 5 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:206]: Removed.

File: pages/careers/_human-friendly.html (hunk 19 above)
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:30]: Fixed in 150196cd4140a538d19d84a38b793b0cbfa64d2a.

File: pages/careers/_human-friendly.html (hunk 17 above)
[the remark this answers is not in the document]
Comment (josh-padnick) [reply pages/careers/_human-friendly.html:26]: Fair enough, fixed in 150196cd4140a538d19d84a38b793b0cbfa64d2a.

File: pages/careers/_open-positions.html (hunk 6 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html:219]: Ok, fair enough. Added in 7b2b16b27cfc26ebc62f1f92a6cf803a732cb26d.

**Comment by josh-padnick** [comment]
Jim, thanks for your great feedback. I've done a final, thorough review to remove any typos and make sure the overall page reads nicely. While part of me was eager to just get this done and move on to my next item, I really like how this page reads now! I'll open a new PR in #334 to merge the changes.
